README.md

```diff
@@ -1951,11 +1951,11 @@ _Libraries for working with various layers of the network._
 - [gnet](https://github.com/fish-tennis/gnet) - `gnet` is a high-performance networking framework,especially for game servers.
 - [gNxI](https://github.com/google/gnxi) - A collection of tools for Network Management that use the gNMI and gNOI protocols.
 - [go-getter](https://github.com/hashicorp/go-getter) - Go library for downloading files or directories from various sources using a URL.
+- [go-multiproxy](https://github.com/presbrey/go-multiproxy) - Library for making HTTP requests through a pool of proxies offering fault tolerance, load balancing, automatic retries, cookie management, and more, via http.Get/Post replacement or http.Client RoundTripper drop-in
```

Comment:
The exported types/functions are missing doc comments.

_CONTRIBUTING.md_
>be thoroughly documented (README, pkg.go.dev doc comments, etc.) in the English language, so everyone is able to understand the project's intention and how it works. All public functions and types should have a Go-style documentation header;

Reply:
Thank you for the quick review. Sorry I missed that, the exported functions work like http.{Get,Post} but its best to have the doc comments specified. Please see commit at: https://github.com/presbrey/go-multiproxy/commit/d8ce296a6a60e1bc6e7dd56d854d680ff5a2e7d0

Reply:
LGTM 🔥 .

I'll merge as soon as the changes are released.

Reply:
Great! Thank you, here is the release:
> https://github.com/presbrey/go-multiproxy/releases/tag/v1.0.1